tests/e2e/performance/test_small_file_workload.py

```diff
@@ -40,40 +42,77 @@ class TestSmallFileWorkload(E2ETest):
     workloads
     """
 
+    def analyze_results(self, results, logs):
+
+        def add_value(res, key, value):
+            if key not in res.keys():
+                res[key] = [value]
+            else:
+                res[key].append(value)
+            return res
+        op = ''
+        for line in logs.split('\n'):
+            log.debug(line)
+            if "operation :" in line:
```

Comment:
@Avilir , try smallfile --output-format=json to make your life easier here.    This kind of CI is exactly why I added that feature to it.

Reply:
@bengland2 you are looking on outdated file, i delete this function, and instead of taking the results from the log file i pull them directly from the ElastcSearch server.